src/theme/colors.js

```diff
@@ -0,0 +1,7 @@
+export const themeColors = {
+  primaryBlue: "#rgb(63, 81, 181)",
+  onHoverBlue: "rgb(38, 56, 147)",
+  darkBlue: "rgb(15, 35, 140)",
+  transparenWhite: "rgba(255, 255, 255, 0.7)",
+};
+// all colors used in the app should be here
```

Comment:
you can improve the comment like so: `//NOTE: all colours used in the app should be here`, if you add a comment for a future job, you can write it like this: `//TODO: something to be done at a later date`. And position the comment before the const